save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/testsuitessources/fetch/TestSuitesSourceFetcher.kt

```diff
@@ -200,9 +200,10 @@ private fun innerTestSuitesSourceFetcher() = FC<TestSuitesSourceFetcherProps> {
                         headers = jsonHeaders,
                         loadingHandler = ::loadingHandler,
                     )
-                        .unsafeMap {
+                        .unsafeMap<List<String>> {
```

Comment:
I guess it should be done in all places where we get a new content to select

Reply:
Done